app/views/admin/custom_fields/hierarchy/items/index.html.erb

```diff
@@ -31,4 +31,6 @@ See COPYRIGHT and LICENSE files for more details.
 
 <%= render(Admin::CustomFields::EditFormHeaderComponent.new(custom_field: @custom_field, selected: :items)) %>
 
-<%= render(Admin::CustomFields::Hierarchy::ItemsComponent.new(custom_field: @custom_field)) %>
+<%= turbo_frame_tag :hierarchy_list, data: { turbo_action: :advance } do %>
```

Comment:
`advance` needed to be added here due to the `Breadcrumbs` not allowing us to pass down tags to the `a` elements.